src/common/containers/landing.js

```diff
@@ -64,12 +64,12 @@ class Landing extends Component {
               Publish your software for
             </HeadingTwo>
             <div className={ `${styles.row}` }>
-              <img className={ styles.brandLogo } src={images.debian} />
-              <img className={ styles.brandLogo } src={images.opensuse} />
-              <img className={ styles.brandLogo } src={images.archlinux} />
-              <img className={ styles.brandLogo } src={images.gentoo} />
-              <img className={ styles.brandLogo } src={images.fedora} />
-              <img className={ styles.brandLogo } src={images.ubuntu} />
+              <img className={ styles.brandLogo } alt="Debian," src={images.debian} />
+              <img className={ styles.brandLogo } alt="openSUSE," src={images.opensuse} />
+              <img className={ styles.brandLogo } alt="Arch Linux," src={images.archlinux} />
+              <img className={ styles.brandLogo } alt="Gentoo," src={images.gentoo} />
+              <img className={ styles.brandLogo } alt="Fedora," src={images.fedora} />
+              <img className={ styles.brandLogo } alt="Ubuntu." src={images.ubuntu} />
```

Comment:
@richmccartney Again, [the HTML spec addresses precisely this point](https://html.spec.whatwg.org/multipage/images.html#a-short-phrase-or-label-with-an-alternative-graphical-representation:-icons,-logos): “If the logo is being used to represent the entity, e.g. as a page heading, the `alt` attribute must contain the name of the entity being represented by the logo. The `alt` attribute must *not* contain text like the word ‘logo’, as it is not the fact that it is a logo that is being conveyed, it’s the entity itself.”